Content/Projectiles/Melee/FireStarterProj.cs

```diff
@@ -10,9 +10,9 @@ internal class FireStarterProjectile : ModProjectile
 
 	public override void SetDefaults()
 	{
-		Projectile.width = 40;
-		Projectile.height = 76;
-		Projectile.aiStyle = 0;
+		Projectile.width = 60;
+		Projectile.height = 60;
```

Comment:
I notice this doesn't match the image size, what determines this? I played with it some but noticed the hitbox didn't really change much

Reply:
The hitbox is constant and doesn't rotate, so going with a square hitbox is better. That being said, the projectile has custom collision, so it's really not important at all.